### PR TITLE
SWP-599

### PR DESCRIPTION
**Pull request by tomaszrondio**

### CODE REVIEW

File: amp/amp-theme/index.html.twig [hunk 1]
```diff
@@ -35,7 +35,7 @@ Available variables:
       {% if media.renditions is iterable %}
           {% for rendition in media.renditions %}
               {% if rendition.name == '770x515' %}
```
Comment (ljubar) [inline amp/amp-theme/index.html.twig:37]: Yeah.. this is actually wrong rendition size (from Liquid). Here we need 1250x600, or 610x380

**Review by ljubar — COMMENTED** [review]
Pls change rendition dimensions

File: amp/amp-theme/index.html.twig (hunk 1 above)
Comment (tomaszrondio) [reply amp/amp-theme/index.html.twig:37]: it does not matter bcs we have all renditions anyway. I would say it could be smaller. Leave it for now.

Comment (ljubar) [reply amp/amp-theme/index.html.twig:37]: aha allright

**Review by ljubar — APPROVED** [review]

**Pull request merged**